tests/e2e/support/helpers.js

```diff
@@ -377,6 +377,7 @@ export const SnackBarError = {
   ILLEGAL_SCUTTLE:
     "You can only scuttle an opponent's point card with a higher rank point card, or the same rank with a higher suit. Suit order (low to high) is: Clubs < Diamonds < Hearts < Spades",
   FROZEN_CARD: 'That card is frozen! You must wait a turn to play it',
+  NOT_IN_GAME: "You can't do that because you're not a player in this game",
```

Comment:
It might make sense to move this object into /fixtures. Same goes for the Card enum. @seriouslysean thoughts?